src/types/student/student.response.ts

```diff
@@ -18,3 +18,12 @@ export type StudentResponse = {
     educationAndExperience: EducationAndExperience;
   };
 };
+
+export type StudentsResponse = {
```

Comment:
tu też jest coś źle.. bo to typ dla pojedynczego studenta - trzeba to jakoś inaczej nazwać :)